dotnet/src/IntegrationTests/Fakes/EmailSkillFake.cs

```diff
@@ -11,7 +11,7 @@ internal sealed class EmailSkillFake
 {
     [SKFunction, Description("Given an email address and message body, send an email")]
     public Task<string> SendEmailAsync(
-        [Description("The body of the email message to send.")] string input,
+        [Description("The body of the email message to send.")] string input = "",
```

Comment:
nit: Use `string.Empty`

Reply:
As this is a parameter we can't use `string.Empty` only const values.